spotify-artist-viewer/Models/SAArtistTableViewController.m

```diff
@@ -14,24 +14,23 @@
 
 
 @interface SAArtistTableViewController ()
-//@property (weak, nonatomic) IBOutlet UISearchBar *searchBar;
+
 
 
 @end
 
 @implementation SAArtistTableViewController
-//connecting this view controller with the Artist object (importing SAArtist, too)
 @synthesize artistArray;
 @synthesize searchResults;
-@synthesize rowNum;
+
 
 
 - (void)viewDidLoad {
     [super viewDidLoad];
+    self.artistArray = [NSMutableArray new];
     [self.searchBar setDelegate:self];
     [self.tableView reloadData];
```

Comment:
You do not need to reload your tableView at this point because nothing has changed about your data source.